client/my-sites/plan-features-2023-grid/hooks/npm-ready/data-store/use-highlighted-features.ts

```diff
@@ -0,0 +1,42 @@
+import {
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE,
+	FEATURE_UNLIMITED_SUBSCRIBERS,
+	FEATURE_PAYMENT_TRANSACTION_FEES_10,
+	FEATURE_PAYMENT_TRANSACTION_FEES_8,
+	FEATURE_PAYMENT_TRANSACTION_FEES_4,
+} from '@automattic/calypso-products';
+import type { PlansIntent } from './use-grid-plans';
+
+/**
+ * Returns an array of features that, if they exist for a
+ * particular Plan on a pricing table, will be highlighted
+ * (ie, moved to the top of the list, and bolded).
+ *
+ * If the specified features don't exist for a particular
+ * Plan, they will be ignored.
+ *
+ * The Logic for processing the highlightedFeatures
+ * lives in the adjacent usePlanFeaturesForGridPlans hook.
+ *
+ * TODO clk: move to plans data store
+ */
+const useHighlightedFeatures = (
```

Comment:
Maybe add a comment clarifying that highlighting only works for WPCOM features for now (not the JP ones)? 🤔

Reply:
Added.